target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -78,7 +78,8 @@ tplink,tl-wr941-v4)
 	;;
 tplink,tl-wr740nd-v4|\
 tplink,tl-wr741nd-v4|\
-tplink,tl-wr842n-v2)
+tplink,tl-wr842n-v2|\
+tplink,tl-wr841-v8)
```

Comment:
Every lines should be in alphabetical order. (Which means wr841 should go above wr842n.)